applications/StructuralMechanicsApplication/custom_elements/spring_damper_element_3D2N.cpp

```diff
@@ -0,0 +1,540 @@
+        rValues[index+1] = acc[1];
+        rValues[index+2] = acc[2];
+        // rValues[index+3] = aacc[0];
+        // rValues[index+4] = aacc[1];
+        // rValues[index+5] = aacc[2];
+        rValues[index+3] = 0.0;
+        rValues[index+4] = 0.0;
+        rValues[index+5] = 0.0;
+    }
+}
+
+//************* STARTING - ENDING  METHODS
+//************************************************************************************
+//************************************************************************************
+
+void SpringDamperElement3D2N::Initialize()
+{
+    KRATOS_TRY;
+    
+    KRATOS_CATCH( "" );
+}
+
+////************************************************************************************
+////************************************************************************************
+
+void SpringDamperElement3D2N::InitializeSolutionStep( ProcessInfo& rCurrentProcessInfo )
+{
+    KRATOS_TRY;
+    
+    KRATOS_CATCH( "" );
+}
+
+////************************************************************************************
+////************************************************************************************
+void SpringDamperElement3D2N::InitializeNonLinearIteration( ProcessInfo& rCurrentProcessInfo )
+{
+
+}
+
+////************************************************************************************
+////************************************************************************************
+
+void SpringDamperElement3D2N::FinalizeNonLinearIteration( ProcessInfo& rCurrentProcessInfo )
+{
+
+}
+
+////************************************************************************************
+////************************************************************************************
+
+void SpringDamperElement3D2N::FinalizeSolutionStep( ProcessInfo& rCurrentProcessInfo )
+{
+    KRATOS_TRY;
+    
+    // Explicit case:
+    this->ClearNodalForces();
+
+    KRATOS_CATCH( "" );
+}
+
+//************* COMPUTING  METHODS
+//************************************************************************************
+//************************************************************************************
+
+void SpringDamperElement3D2N::CalculateLocalSystem( MatrixType& rLeftHandSideMatrix, VectorType& rRightHandSideVector, ProcessInfo& rCurrentProcessInfo )
+{
+
+    KRATOS_TRY;
+
+    /* Calculate elemental system */
+    // Compute RHS (RHS = rRightHandSideVector = Fext - Fint)
+    this->CalculateRightHandSide(rRightHandSideVector, rCurrentProcessInfo);
+
+    // Compute LHS
+    this->CalculateLeftHandSide(rLeftHandSideMatrix, rCurrentProcessInfo);
+
+    KRATOS_CATCH( "" );
+}
+
+//***********************************************************************************
+//***********************************************************************************
+
+void SpringDamperElement3D2N::CalculateRightHandSide(VectorType& rRightHandSideVector, ProcessInfo& rCurrentProcessInfo )
+{
+    // std::cout << "CalculateRightHandSide for id=" << this->Id() << std::endl;
```

Comment:
I recommend to do some more code cleaning.
You can still recover these by restoring the development branch, should you be needing them later